api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -134,12 +136,14 @@
 
   @VisibleForTesting
   public void setDistinctWorkspaceLimit(int limit) {
-    distinctWorkspacelimit = limit;
+    distinctWorkspaceLimit = limit;
   }
 
   @Override
   public ResponseEntity<WorkspaceResource> updateRecentResource(
       String workspaceNamespace, String workspaceId, RecentResourceRequest recentResourceRequest) {
+    workspaceAuthService.enforceWorkspaceAccessLevel(
```

Comment:
Is this the right access level to enforce (and for delete)? @als364

I think this only gets called on a save notebook operation so I set the ACL for this to be writer.

Reply:
Yeah, writer should be fine for both of these - I believe we allow notebook deletes for writers